File: backend/src/main/java/com/backend/ord/domain/dto/abstracts/DTOBase.java
package com.backend.ord.domain.dto.abstracts;

import lombok.Getter;
import lombok.Setter;

import java.time.Instant;
import java.util.UUID;

@Getter
@Setter
public abstract class DTOBase {
    protected UUID id;
    protected Instant createdAt;
    protected Instant updatedAt;
}

File: backend/src/main/kotlin/com/backend/ord/domain/dto/abstracts/DTOBase.kt
package com.backend.ord.domain.dto.abstracts

import java.time.Instant
import java.util.*

abstract class DTOBase(
    val id: UUID = UUID.randomUUID(),
    val createdAt: Instant = Instant.now(),
    var updatedAt: Instant = Instant.now(),
)
